src/pages/Contributors.js

```diff
@@ -53,200 +60,58 @@ const Contributors = () => {
-  } else {
-    const {
-      name,
-      twitter_username,
-      avatar_url,
-      blog,
-      html_url,
-    } = contributorData
-    console.log(blog)
-    return (
-      <div className='flex items-center px-4 py-6 text-center bg-white shadow border-l-4 border-teal-500 transition duration-150 ease-in-out'>
-        <div className='flex flex-col ml-2 items-center justify-between'>
-          <img
-            className='rounded-full h-24 w-24 mx-auto'
-            src={avatar_url}
-            alt={`${name} avatar`}
-          />
-        </div>
-        <div className='flex flex-col ml-6 items-start'>
-          {name ? (
-            <span className='font-semibold mb-2 text-blue-900'>{name}</span>
-          ) : (
-            <span className='font-semibold mb-2 text-blue-900'>
-              {contributor.login}
-            </span>
-          )}
-          <div className='flex items-center mb-2'>
-            {twitter_username && (
-              <a
-                href={`https://twitter.com/${twitter_username}`}
-                className='flex mr-4 items-center text-teal-500 hover:text-teal-800 transition-colors ease-in-out duration-75 w-auto'
-                target='_blank'
-                rel='noopener noreferrer'
-              >
-                <svg
-                  className='fill-current flex-shrink w-4'
-                  aria-hidden='true'
-                  focusable='false'
-                  data-prefix='fab'
-                  data-icon='twitter'
-                  role='img'
-                  xmlns='http://www.w3.org/2000/svg'
-                  viewBox='0 0 512 512'
-                >
-                  <path
-                    fill='currentColor'
-                    d='M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z'
-                  ></path>
-                </svg>
-              </a>
-            )}
+  return (
+    <div className='flex items-center px-4 py-6 text-center bg-white shadow border-l-4 border-teal-500 transition duration-150 ease-in-out'>
+      <div className='flex flex-col ml-2 items-center justify-between'>
+        <img
+          className='rounded-full h-24 w-24 mx-auto'
+          src={avatar_url}
+          alt={`${name} avatar`}
+          loading='lazy'
```

Comment:
Added the `loading="lazy"` attribute to improve page load. See [Native image lazy-loading for the web!](https://addyosmani.com/blog/lazy-loading/).